teos/src/tls.rs

```diff
@@ -5,19 +5,19 @@
  * https://github.com/ElementsProject/lightning/blob/master/LICENSE
 */
 
-use rcgen::{Certificate, KeyPair, RcgenError};
+use rcgen::{Certificate, Error, KeyPair};
```

Comment:
nit: I would got for renaming `Error as RcgenError` here for more clarity in the usage down there.

Reply:
Done